### PR TITLE
Fix issue with check button in dashboard

### LINKED ISSUE #3703: Weird behavior of the `Run Linkerd Check` button
I've just checked out master, cleaned up the browser's cache, launched the dashboard, and when I tried to click the `run linkerd check` button it jumps to the left, and I have to click on it again:

ezgif com-video-to-gif

Using Chrome, under Linux

### PR DESCRIPTION
**Pull request by cynthia-sg**
This PR removes `Linkerd run check` button wrapper from CheckModal component and cleans styles

Fixes #3703 

Signed-off-by: Cintia Sanchez Garcia <cynthiasg@icloud.com>

### CODE REVIEW

**Comment by cynthia-sg** [comment]
Hi @alpeb , I have not been able to reproduce the issue (using Ubuntu 18 and Chrome 78.0.3904.97). I have removed the button wrapper that could be causing the issue. Could you please try it? If it’s still failing, could you please give me some more details about your browser version and extensions?

**Comment by alpeb** [comment]
@cynthia-sg I still see the same issue. I'm on Chrome 76.0.3809.132 and these are my extensions:
image

**Comment by cynthia-sg** [comment]
Hi @alpeb, I think this issue is related to this bug `https://bugs.chromium.org/p/chromium/issues/detail?id=992010`, I have pushed a change that should solve the problem (tested successfully in Chromium 76.0.3809.0). Could you please let me know if yours is working too?

Thanks

**Comment by alpeb** [comment]
@cynthia-sg it appears you forgot to push? I can't see the new commit.

**Comment by cynthia-sg** [comment]
Yes, you are right, sorry about that @alpeb.

File: web/app/js/components/ServiceMesh.jsx [hunk 1]
```diff
@@ -294,4 +304,4 @@ class ServiceMesh extends React.Component {
   }
 }
 
-export default withContext(ServiceMesh);
+export default withStyles(styles)(withContext(ServiceMesh));
```
Comment (scottcarol) [inline web/app/js/components/ServiceMesh.jsx]: Mind fixing the newline warning?

**Pull request merged**